test/e2e/setup.js

```diff
@@ -1,4 +1,4 @@
-const { Builder, By } = require('selenium-webdriver');
+const { Builder, By, Key } = require('selenium-webdriver');
```

Comment:
Beter, inderdaad :)

We zouden die By en Key wel beter exporteren vanuit een andere file, zo voorkomen we dat de core config en setup telkens geimporteerd moet worden in component objecten. 
Maar dat is dan weer een breaking change. Misschien dat best laten voor een ander moment.

Reply:
+1, nog eens bekijken hoe dat zat met het probleem met de test-setup bij impactscore en het niet slechter maken op zijn minst

Reply:
@SanderKleykens lijkt me inderdaad een goed voorstel, maak hiervoor gerust een issue aan.

@pascalriquier ik dacht dat daarvoor al een issue bestond, anders mag je dat gerust ook nog aanmaken.

@kevcraey ter info.

Reply:
Beide opmerkingen zijn scope van andere issues, dus resolve in kader van dit issue.